static/pages/home/home.html

```diff
@@ -340,28 +324,23 @@ <h3 class="md-display-1">
            layout-fill>
     <md-card-content class="home-page-display-size
                             oppia-foundation-no-padding"
-                     layout="column"
-                     layout-align="center center">
+                      layout="column" 
+                      layout-align="center center">
       <md-content class="oppia-foundation-black-text
-                         oppia-foundation-white-background
-                         home-page-description-text-box"
```

Comment:
Class deleted by accident?

Reply:
I guess I misunderstood your feedback earlier: https://github.com/oppia/foundation-website/pull/41#discussion_r216429304 

I thought that the class needed to be deleted.

Reply:
This? https://github.com/oppia/foundation-website/pull/41#discussion_r216429304
Hm, can you check if there is any visual changes with and without that css class?

Reply:
Yes, that's what I'm referring to. There doesn't appear to be any difference.

Reply:
Are you sure?
![screen shot 2018-09-11 at 2 00 33 pm](https://user-images.githubusercontent.com/11153258/45378320-2182aa00-b5cb-11e8-8eb1-2e0fa6416940.png)

vs

![screen shot 2018-09-11 at 2 00 17 pm](https://user-images.githubusercontent.com/11153258/45378330-26475e00-b5cb-11e8-8be9-605e92e27a2f.png)